extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -608,10 +608,15 @@ export class AmpAnalytics extends AMP.BaseElement {
       for (const k in this.config_['requests']) {
         if (hasOwn(this.config_['requests'], k)) {
           const request = this.config_['requests'][k];
-          requests[k] = new RequestHandler(
-              this.getAmpDoc(), request, this.preconnect,
-              this.sendRequest_.bind(this),
-              this.isSandbox_);
+          try {
+            requests[k] = new RequestHandler(
```

Comment:
i think we can leave it thrown, it's a user error that should be fixed.

Reply:
I don't know. I feel it's better to scope the error within the request.

Reply:
we should not cover the error. it might not easily discoverable. let it throw and break the page.

Reply:
Sounds good, since `batchPlugin` is a new supported keyword, and we want to be strict on that. Fixed